persistence/src/main/java/com/epam/esm/model/Certificate.java

```diff
@@ -0,0 +1,265 @@
+package com.epam.esm.model;
+
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.*;
+
+import static java.util.Objects.hash;
+import static java.util.Objects.requireNonNull;
+
+
+@Entity
+@Table(name = "certificate")
+public class Certificate implements Model {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "description")
+    private String description;
+    @Column(name = "price")
+    private Integer price;
+    @Column(name = "duration")
+    private Integer duration;
+    @Column(name = "create_date")
```

Comment:
@Column annotation has params to describe column or set field behavior more precisely. Check for example "updatable" for create_date case. It would be great if you describe you columns using nullable, length etc.

Additionally check annotations @PrePersist, @PreUpdate it may be useful to handle timestamps for entity.